include/constants.h

```diff
@@ -81,7 +81,7 @@ static const char *LINSYS_SOLVER_NAME[] = {
 # endif /* ifndef OSQP_NULL */
 
 # ifndef OSQP_NAN
-#  define OSQP_NAN ((c_float)0.0/0.0) // Not a Number
+#  define OSQP_NAN ((c_float)0x7fc00000UL) // Not a Number
```

Comment:
It makes sense. I found that solution on stackoverflow but apparently it is not so portable. It worked on my laptop. Did it give you issues on Windows? Unless we use C99, this might be not a portable solution see [this](https://stackoverflow.com/questions/6235847/how-to-generate-nan-infinity-and-infinity-in-ansi-c).

Reply:
It works on Windows when I compile it with MinGW, but I get an error when compiling the Python interface.